muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json

```diff
@@ -490,6 +490,12 @@
       "updateError": "$t(common:notifications.updateError)",
       "updateError_matriculationEligiblity": "Loading the eligibility rights for the {{subject}} failed. Please try refreshing the page.",
       "updateError_summary": "Loading the summary information failed. Please try refreshing the page."
+    },
+    "wcag": {
+      "excersiseActivity": "Harjoitustehtäviä tehty {{percent}}%",
+      "evaluableActivity": "Arvioitavia tehtäviä tehty {{percent}}%",
```

Comment:
EN locales missing in lines 495 and 496 and might be good to change the keys also to be more inline with our terminology.
```
"exercisesActivity": "Exercises done {{percent}}%",
"assignmentsActivity": "Assignments done {{percent}}%",
```